bin/node-template/pallets/template/src/lib.rs

```diff
@@ -14,8 +14,20 @@ mod tests;
 #[cfg(feature = "runtime-benchmarks")]
 mod benchmarking;
 pub mod weights;
+
+mod storages;
+mod events;
+mod errors;
+mod calls;
```

Comment:
These *are* modules, but i am not sure if anyone actually wants to use them as normal Rust modules?  
Maybe we can find a more FRAME syntax which just wraps these and the `import_sections` below. Although i would definitely keep this syntax working with the manual mods and manual `import_sections`.

For example:  
```rust
#[pallet_sections(
    mod storages;
    mod events;
    mod errors;
    mod calls;
)]
```
(not quite happy with the syntax, but you see what i mean)